packages/scripts/config/webpack.config.js

```diff
@@ -228,6 +229,8 @@ const config = {
 		// obsolete and should be removed. Related webpack issue:
 		// https://github.com/webpack-contrib/mini-css-extract-plugin/issues/85
 		new FixStyleWebpackPlugin(),
+		// OptimizeCssAssetsPlugin to minify CSS
+		// new OptimizeCssAssetsPlugin(),
```

Comment:
It looks like it was accidentally commented out 😄

Reply:
LOL, was working with the other repo to test this out, accidentally pushed the wrong one 🤣
BTW, it makes sense to put it in the `plugins` array instead for consistency since it's a plugin as well - thanks for pointing it out. Any idea why adding it to the `minimizer` array works the same?

Reply:
It's hard to tell. I noticed that there is also an official plugin from webpack (v5) that integrates with the `minimizer` so maybe it's related to that.